src/Translator/OpenAiTranslator.php

```diff
@@ -114,4 +122,9 @@ public function isConfigured(): bool
 
         return $openAiConfig['api_key'] && $openAiConfig['model'];
     }
+
+    private function CustomPrompt(): string
```

Comment:
Method should be in camelcase `customPrompt()` 😄

Reply:
fixed :  ) @negoziator